### DISCUSSION TITLE
How does search work?

### DISCUSSION THREAD

**Discussion by moonseppel** [jonaswinkler/paperless-ng#1087] [Support] [answered]
Hi,

I have set up perpless-ng, scanned some documents and ma very happy with how it works so far. But today I stumbled upon something irritating. I have a document that has "Haus-/ Wohnungsübergabe" in its content. I searched for "übergabe" and yet did not find the mentioned document, though paperless does search in the content, because it did find another document with "Übergabe" in its content. Does it not search fro word segements by default? Can I add like wildcards to my search term (e.g. "*übergabe*") to find the document? Didn't find anything about how search works in the docs. Maybe you can provide some help.

**Comment by jonaswinkler** [comment, marked as answer]
Searching is documented here: https://paperless-ng.readthedocs.io/en/latest/usage_overview.html#searching

**Reply by moonseppel** [reply]
Thank you :) I somehow overlooked it. Does not exactly mention my use case, but "*übergabe*" does the trick. Maybe that would be something to add there?

**Reply by jonaswinkler** [reply]
Maybe "Matching inexact / partial words:" instead, and a couple more examples. Would you mind creating a PR?

**Reply by moonseppel** [reply]
Sure :) I tired this one https://github.com/jonaswinkler/paperless-ng/pull/1089 This is my first PR ever on Github, so hopefully I did it right.

And thanks also for the quick replies :)
